packages/fxa-content-server/app/scripts/lib/auth-errors.js

```diff
@@ -287,7 +287,8 @@ var ERRORS = {
   },
   INVALID_EXPIRED_SIGNUP_CODE: {
     errno: 183,
-    message: t('Invalid or expired verification code'),
+    message: t('Invalid or expired confirmation code'),
+    version: 2,
```

Comment:
I don't think we actually need `version: 2` here since this is using gettext (you can always tell if there's a helper `t` or `unsafeTranslate` function in the JS or mustache files). Gettext uses the English string as the l10n ID while Fluent explicitly requires an ID (goes by key/value pairs).

Reply:
OH I get it thank you!!